server/utils/emulator.go

```diff
@@ -16,23 +16,21 @@
  * limitations under the License.
  */
 
-package server
+package utils
```

Comment:
why do you see moving this to utils to be better? Generally I don't like packages like `utils`, `helpers`, `misc` etc. Also not just my opinion https://rakyll.org/style-packages/

Reply:
I am fan of Util naming here too tbh, but I couldn't find a package name covering : admin, liveness and emulator API. Maybe `management` ?